scripts/main/lychee.js

```diff
@@ -744,6 +972,12 @@ lychee.load = function (autoplay = true) {
 			if (visible.sidebar()) sidebar.toggle(false);
 			mapview.open();
 			lychee.footer_hide();
+		} else if (albumID === "frame") {
+			if (lychee.mod_frame_enabled) {
+				frame.initAndStart();
+			} else {
+				loadingBar.show("error", "Frame mode disabled");
```

Comment:
We are inconsistent about that (as always). We localize many error messages but it is far from being complete. In particular, strings which come from the "view" and the "frame" mode (as this one) are mostly not localized, probably because these two modes have always been the unbeloved children.

Here, we have a comparable issue like in other places, too. I only moved this code from the former frame mode into the unified frontend, hence this code appears in the diff. I could have fixed the localization, but I did not feel like doing so. Otherwise I also should have fixed dozens of other strings, too, for the sake of consistency. Maybe, this should be an independent PR where we step over all strings (it shouldn't be too difficult to `grep` for then) and then add localizations for all of them.

What do you want? Do it now? Do it later?

Reply:
Well, neither the view nor the frame mode supported localization as far as I know, so no surprise.

I thought @qwerty287 did such a grep relatively recently and added the missing strings he could find?

I'm generally for trying not to add more issues with the code we add. But, as you said, this is only the first PR of a series. So if you address this in one of the subsequent PRs, I'm OK with that.

Reply:
Personally, I don't feel like addressing it. (I hate these "cleaning up" jobs.) But yes, I probably will. I just did a grep for `loadingBar.show\("\w+", "|\.textContent = "` and found four untranslated strings. Probably, there are some more if one searches for other keywords as well.

Yes, I will do that in another PR.